content/docs/concepts/traffic-management/traffic-routing/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Traffic Routing
+description: Describes traffic routing on Istio using service subsets and provides the most common uses.
+weight: 8
+keywords: [routing, request, traffic routing, requests routing, A/B testing, splitting, circuit breakers, subsets, subset]
+aliases:
+    - /docs/concepts/traffic-management/pilot
+---
+
+As described in [Pilot's architecture](../pilot), Pilot maintains an
+`Abstract Model` with the canonical representation of the services in a mesh.
+Istio's model of a service is independent of its representation in the
+underlying platform: Kubernetes, Mesos, Cloud Foundry, etc. The
+platform-specific adapters populate the `Abstract Model` representation with
+various fields from the metadata in the platform.
+
+To cover common configuration scenarios, Istio introduces the concept of a
+**service subset**, to subdivide service instances by versions, say `v1` and
```

Comment:
**service subset**. Defining subsets of services lets you subdivide your service instances by versions....